x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/InstanceType.java

```diff
@@ -62,7 +62,9 @@ public static InstanceType fromHostSource(Map<String, Object> source) {
         if (region != null) {
             // example: "azure.compute.location": "eastus2"
```

Comment:
Nit: Maybe move this outside the `if` and closer to where `azure.compute.location` is read?